discopop_explorer/__main__.py

```diff
@@ -23,10 +23,12 @@
     --fmap=<fmap>               File mapping [default: FileMapping.txt]
     --json=<json_out>           Json output
     --plugins=<plugs>           Plugins to execute
-    --cu-inst-res=<cuinstres>   CU instantiation result file. Task Pattern Detector is executed if this option is set.
+    --task-pattern              Enables the Task Pattern Detection.
```

Comment:
don't use camel case plz.
"Enables the task parallelism pattern identification,"

Reply:
should be fixed now.